<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CONFIRMATION</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="pic1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>-20</y>
      <width>801</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(photo/tombA.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>691</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Bebas&quot;;
background-image: url(photo/bg4.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 21pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string> price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>610</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 21pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string> card  number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 14pt &quot;Granite&quot;;</string>
    </property>
    <property name="text">
     <string>PAYMENT  INFORMATION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>660</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 21pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string> security  code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 14pt &quot;Granite&quot;;</string>
    </property>
    <property name="text">
     <string>TOMB  INFORMATION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>971</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(photo/redbg2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 21pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string> tomb  number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="showtombnumber">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 25pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="showprice">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>410</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 25pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="getnewcvv">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>660</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 20pt &quot;Wild Zova Free&quot;;
background-color: rgb(79, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="getnewcardnumber">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>610</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 20pt &quot;Wild Zova Free&quot;;
background-color: rgb(79, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="getname1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>460</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 20pt &quot;Wild Zova Free&quot;;
background-color: rgb(79, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="getname2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>500</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 20pt &quot;Wild Zova Free&quot;;
background-color: rgb(79, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="second">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 21pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string> 2  person  name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 25 21pt &quot;Wild Zova Free&quot;;</string>
    </property>
    <property name="text">
     <string> 1  person  name</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>781</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(photo/tombB.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>730</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 11pt &quot;Granite&quot;;
background-color: rgb(79, 0, 0);</string>
    </property>
    <property name="text">
     <string>CONFIRM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>730</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 191, 0);
font: 11pt &quot;Granite&quot;;
background-color: rgb(79, 0, 0);</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QLabel" name="popmessage">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>700</y>
      <width>691</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>pic2</zorder>
   <zorder>label_2</zorder>
   <zorder>pic1</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_9</zorder>
   <zorder>showtombnumber</zorder>
   <zorder>showprice</zorder>
   <zorder>getnewcvv</zorder>
   <zorder>getnewcardnumber</zorder>
   <zorder>getname1</zorder>
   <zorder>getname2</zorder>
   <zorder>second</zorder>
   <zorder>label_12</zorder>
   <zorder>confirm</zorder>
   <zorder>back</zorder>
   <zorder>popmessage</zorder>
  </widget>
 </widget>
 <resources>
  <include location="photo/convert.qrc"/>
 </resources>
 <connections/>
</ui>
